modules/core/src/test/java/org/apache/ignite/internal/processors/cache/CacheGroupsMetricsRebalanceTest.java

```diff
@@ -192,25 +194,23 @@ public void testRebalance() throws Exception {
         final CountDownLatch l1 = new CountDownLatch(1);
```

Comment:
I would prefer meaningful names: `startRebalanceLatch` and `blockRebalanceLatch` for instance.